<?xml version="1.0"?>
<launch>

    <param name="simulation_mode" type="bool" value="true" />

    <!-- Soccer Gazebo Simulator -->
    <include file="$(find soccersim)/launch/soccer.launch">
        <arg name="vision_home1" value="/vision_ally1_position" />
        <arg name="vision_home2" value="/vision_ally2_position" />
        <arg name="vision_away1" value="/vision_opponent1_position" />
        <arg name="vision_away2" value="/vision_opponent2_position" />
        <arg name="vision_ball" value="/vision_ball_position" />
    </include>

    <!-- Ally1 -->
    <group ns="/ally1">
        <include file="$(find playground)/launch/hw_agnostic.launch">
            <arg name="me" value="ally1" />
            <arg name="ally" value="ally2" />
            <arg name="vel_cmds_remap" value="/home1/command" />
        </include>
    </group>

</launch>